src/OVAL/probes/unix/linux/rpmverifypackage.c

```diff
@@ -43,29 +43,16 @@
 #include <fcntl.h>
 #include <pcre.h>
 
-/* RPM headers */
-#include <rpm/rpmdb.h>
-#include <rpm/rpmlib.h>
-#include <rpm/rpmts.h>
-#include <rpm/rpmmacro.h>
-#include <rpm/rpmlog.h>
+#define RPM_NOT_TOUCH
+#include "rpm-helper.h"
+
+/* Individual RPM headers */
 #include <rpm/rpmfi.h>
-#include <rpm/header.h>
 #include <rpm/rpmcli.h>
 #include <popt.h>
 
-#ifndef HAVE_HEADERFORMAT
-# define HAVE_LIBRPM44 1 /* hack */
-# define headerFormat(_h, _fmt, _emsg) headerSprintf((_h),( _fmt), rpmTagTable, rpmHeaderFormats, (_emsg))
-#endif
-
-#ifndef HAVE_RPMFREECRYPTO
-# define rpmFreeCrypto() while(0)
-#endif
-
-#ifndef HAVE_RPMFREEFILESYSTEMS
-# define rpmFreeFilesystems() while(0)
-#endif
+#define RPM_NOT_TOUCH
```

Comment:
Original source code was disabling rpmFreeCrypto(),.. by macros. From some reason the disable code wasn't for this one function. I can investigate log of previous commits